modules/virt-storage-wizard-fields-web.adoc

```diff
@@ -26,3 +26,36 @@
 |The `StorageClass` that is used to create the disk.
 
 |===
+
+[id="virt-storage-wizard-fields-advanced-web_{context}"]
+[discrete]
+== Advanced storage settings
+The following advanced storage settings are available for *Blank*, *URL*, and *Attach Cloned Disk* disks.
+These parameters are optional. If you do not specify these parameters, the defaults are used, based on what has been configured in the `kubevirt-storage-class-defaults` ConfigMap.
```

Comment:
Just noting that if you opt to change the wording of this sentence in the previous module, you should change it again here.